common/util/file_util.cc

```diff
@@ -46,6 +48,39 @@ absl::string_view Stem(absl::string_view filename) {
              : filename.substr(0, last_dot_pos);
 }
 
+bool UpwardFileSearch(absl::string_view start, absl::string_view filename,
+                      std::string& result) {
+  /* Convert to absolute path */
+  char absolute_path[PATH_MAX];
+  if (realpath(std::string(start).c_str(), absolute_path) < 0) {
```

Comment:
`start.data()` is also a const char*, no need to copy to std::string.

Reply:
absolutely right, thanks!